src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution.cs

```diff
@@ -2574,7 +2574,7 @@ static TypeSymbol getParameterTypeAndRefKind(int i, MemberAnalysisResult memberR
 
                 var type = parameter.Type;
                 if (memberResolutionResult.Kind == MemberResolutionKind.ApplicableInExpandedForm &&
-                    parameter.Ordinal == parameters.Length - 1)
```

Comment:
>.Ordinal [](http://example.com/codeflow?start=29&length=8)

Consider reviewing other places where the ```Ordinal``` property is used in the codebase and confirming that no adjustments necessary with respect to extensions.

Reply:
I reviewed all the usages of `Ordinal`. Added some tests for that in last commit and fixed handling of `this` modifier.